bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/renovator/MovingAsset.java

```diff
@@ -61,6 +67,14 @@ public void move(ReplicatorQueue replicatorQueue, ResourceResolver rr) throws Il
             session.getWorkspace().getObservationManager().setUserData("changedByWorkflowProcess");
             session.move(getSourcePath(), getDestinationPath());
             session.save();
+            if (session.nodeExists(getDestinationPath())) {
```

Comment:
```suggestion
            if (session.nodeExists(getDestinationPath() + "/" + JcrConstants.JCR_CONTENT)) {
```

WDYT about that just to be ultra-safe? If were doing the check why not check exactly for the node were writing to?